versions/linea-mainnet.env

```diff
@@ -1,4 +1,4 @@
-LINEA_BESU_TAR_GZ=25.2-delivery50
+LINEA_BESU_TAR_GZ=25.2.0-linea1
```

Comment:
let's not change this until other envs have been tested

Reply:
Reverted.